### PR TITLE
Clean

### PR DESCRIPTION
**Pull request by kahyunnam**
### Summary
- delete duplicates created while organizing folders a while ago + delete useless files in general
- error handling for running main script + now takes in ip address as bash arg so that we don't have to manually change the hard coded in address

### Testing
experimental testing

### Notes
- would be better to just configure ground pi to have a static ip address instead of this bash arg solution. to be looked into later

### Comments
- [x] Add an x between the brackets if you commented your code well!

### CODE REVIEW

File: UDP_groundstation/main.py [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+import socket
 
 if __name__ == "__main__":
-    server = ServerTester("192.168.0.101", 3333)
-    server.listen_for_data()
\ No newline at end of file
+
+    if (len(sys.argv) == 2):
+        udp_addr = "192.168.0." + str(sys.argv[1])
+        print(f"trying to connect to {udp_addr}\n")
```
Comment (tmf97) [inline UDP_groundstation/main.py:10]: nit: replace all print statements with logging statements:
```python
import logging

logging.info("")
logging.warning("")
logging.debug("")
logging.error("")
```

**Review by tmf97 — COMMENTED** [review]
Solid stuff, thank you very much for making this nice a clean!

I'll approve this once we have a README in the UDP_groundstation folder specifying how the high-level pipeline works, and how to get stuff running on the cislunar-hitl pi. We want to be able to re-setup all this infrastructure in case we get newer hardware or something breaks. (I know you know how to do this, but if it's not written down anywhere...)

For the static IP, i've heard it's bad practice and we should use DHCP reservations on the router instead: https://serverfault.com/questions/544619/static-ip-vs-dhcp-reservation

**Comment by kahyunnam** [comment]
will do! thanks for the feedback

**Comment by kahyunnam** [comment]
@tmf97 I changed to logging statements + added a README

Since this repository only contains work for one step of the data pipeline, the README only has an overview + instructions for that step

But I did include a pic to give an idea of how this step connects to others

**Review by tmf97 — APPROVED** [review]
Solid stuff!

**Pull request merged**